deployment/k8s/helmfile/values-dev.yaml

```diff
@@ -0,0 +1,172 @@
+conditionalPackageInstalls:
+  snapshotVolume:
+    enabled: true
+  sharedConfigs:
+    enabled: true
+  migrationConsole:
+    enabled: true
+  bulkLoader:
+    enabled: true
+  kafka:
+    enabled: true
+  replayer:
+    enabled: true
+  captureProxy:
+    enabled: true
+  jaeger:
+    enabled: false
+  prometheus:
+    enabled: false
+  grafana:
+    enabled: false
+  source:
+    enabled: true
+  target:
+    enabled: true
+
+sharedConfigs:
+  globalParameters:
+    sourceCluster:
+      allowRuntimeOverride: true
+      object:
+        endpoint: "http://sourcecluster.example.com:9200"
+        allowInsecure: true
+    kafkaBrokers:
+      allowRuntimeOverride: false
+      object:
+        brokers: "captured-traffic-kafka-bootstrap.ma.svc:9092"
+
+
+
+captureProxy:
+  parameters:
+    destinationUri:
+      source: otherConfig
+      configMapName: "source-cluster"
+      configMapKey: "endpoint"
+    insecureDestination:
+      source: otherConfig
+      configMapName: "source-cluster"
+      configMapKey: "allowInsecure"
+    kafkaConnection:
+      source: otherConfig
+      configMapName: "kafka-brokers"
+      configMapKey: "brokers"
+
+
+migrationConsole:
+  snapshotVolumeEnabled: true
```

Comment:
would be nice if we could have a single-source of truth for this value since it's shared in at least 2 other places.  Same for other occurrences of the pattern.  Is that possible w/ a helmfile?